src/test/java/core/basesyntax/dao/FileWriterImplTest.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.dao;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    public static final String CORRECT_PATH = "src/test/java/resources/output_test.csv";
+    public static final String INCORRECT_PATH = "";
+    private static FileWriterImpl writerService;
+
+    @BeforeClass
+    public static void setUp() {
+        FileWriter fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    public void write_correctPath_ok() {
+        FileWriter fileWriter = new FileWriterImpl();
+        Assert.assertTrue("File is written!", fileWriter.write(CORRECT_PATH, ""));
```

Comment:
we should write and check if the data is in the file.
is data actually written in file?
do you append or rewrite the content?